jpa/subsystem/src/main/java/org/jboss/as/jpa/config/PersistenceUnitMetadataImpl.java

```diff
@@ -379,6 +379,10 @@ public boolean needsJPADelegatingClassFileTransformer() {
 
     @Override
     public void addTransformer(ClassTransformer classTransformer) {
+        if(classTransformer.getClass().getName().contains("org.hibernate")) {
+            ROOT_LOGGER.infof("ignoring entity class transformer from Hibernate ORM as we will instead use a WildFly prototype");
```

Comment:
This needs to either log a a different level, likely `DEBUG`, or it needs o be internationalized.

Reply:
Agreed, that slipped through :)